jenkins-scripts/dsl/sdformat.dsl

```diff
@@ -378,14 +378,11 @@ all_branches.each { branch ->
       }
 
       if (branch == 'sdformat4')
-        ign_math_v="2"
-      else
-        ign_math_v="3"
+        disabled()
 
       steps {
         batchFile("""\
               set USE_IGNITION_ZIP=FALSE
-              set IGNMATH_BRANCH=ign-math${ign_math_v}
```

Comment:
Just checking: how are versions 6+ getting `ign-math`?

Reply:
the [sdformat-default-devel-windows-amd64.bat](https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/sdformat-default-devel-windows-amd64.bat#L10-L14) script calls a different build script for sdformat6+ than for earlier versions. The `IGNMATH_BRANCH` variable is not used by sdformat6+, and since we've disabled the windows build for all earlier versions; I've just removed the variable here